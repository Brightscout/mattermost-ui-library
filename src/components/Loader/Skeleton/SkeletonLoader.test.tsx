import React from 'react';
import {shallow, ShallowWrapper} from 'enzyme';

import SkeletonLoader from '.';

describe('Skeleton Loader', () => {
    let component: ShallowWrapper<any, Readonly<{}>, React.Component<{}, {}, any>>;

    beforeEach(() => {
        component = shallow(<SkeletonLoader className='skeleton-loader-class' />);
    });

    it('Should render correctly', () => {
        expect(component).toMatchSnapshot();
    });

    it('Should apply the passed className prop', () => {
        expect(component.hasClass('skeleton-loader-class')).toBeTruthy();
    });
});
