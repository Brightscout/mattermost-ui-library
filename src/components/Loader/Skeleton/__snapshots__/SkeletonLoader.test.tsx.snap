// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Skeleton Loader Should render correctly and matches snapshot 1`] = `ShallowWrapper {}`;
